src/components/SimpleChart/ChartHeader.tsx

```diff
@@ -0,0 +1,84 @@
+import type { FlexProps } from '@chakra-ui/react'
+import { Flex, Heading } from '@chakra-ui/react'
+import { bnOrZero } from '@shapeshiftoss/chain-adapters'
+import type { UTCTimestamp } from 'lightweight-charts'
+import type { ReactElement, ReactNode } from 'react'
+import styled from 'styled-components'
+import { Row } from 'components/Row/Row'
+import { useLocaleFormatter } from 'hooks/useLocaleFormatter/useLocaleFormatter'
+
+import { useHeaderDateFormatter } from './hooks'
+
+const ChartHeaderWrapper = (props: FlexProps) => (
+  <Flex position='absolute' width='full' gap={4} alignItems='flex-start' zIndex='4' {...props} />
+)
+const ChartHeaderLeftDisplay = styled.div`
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  gap: 4px;
+  padding-bottom: 14px;
+  text-align: left;
+  pointer-events: none;
+  width: 70%;
+
+  * {
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+`
+
+interface HeaderValueDisplayProps {
+  /** The number to be formatted and displayed, or the ReactElement to be displayed */
+  value?: number | ReactElement
+}
+
+function HeaderValueDisplay({ value }: HeaderValueDisplayProps) {
+  const {
+    number: { toFiat },
+  } = useLocaleFormatter()
+
+  if (typeof value !== 'number' && typeof value !== 'undefined') {
+    return <>{value}</>
+  }
+
+  return (
+    <Heading as='h3' lineHeight={1}>
+      {toFiat(bnOrZero(value).toString())}
+    </Heading>
+  )
+}
+
+interface HeaderTimeDisplayProps {
+  time?: UTCTimestamp
+  /** Optional string to display when time is undefined */
+  timePlaceholder?: string
```

Comment:
Do we need this? AFAIK if `time` is undefined, then sorting should be rugged and the chart should throw

Reply:
Not whats its used for, when we get undefined that means they aren't scrubbing the chart.